tests/unit/test_client.py

```diff
@@ -1477,9 +1652,16 @@ def test_create_table_w_fully_qualified_string(self):
         client = self._make_one(project=self.PROJECT, credentials=creds)
         resource = self._make_table_resource()
         conn = client._connection = make_connection(resource)
-
-        got = client.create_table(
-            "{}.{}.{}".format(self.PROJECT, self.DS_ID, self.TABLE_ID)
+        with mock.patch(
+            "google.cloud.bigquery.opentelemetry_tracing._get_final_span_attributes"
+        ) as final_attributes:
+            got = client.create_table(
+                "{}.{}.{}".format(self.PROJECT, self.DS_ID, self.TABLE_ID)
+            )
+        final_attributes.assert_called_once_with(
```

Comment:
style nit: most of the other tests have a blank line before `final_attributes.assert_called_once_with`. Please be consistent.